### ISSUE TITLE
Getting rid of "This page is not secure" warning on some mobile browsers

### ISSUE THREAD

**Issue by Ace2213** [humhub/humhub#3636]
I implemented site-wide static SSL encryption yesterday, and everything was fine. One user reported a "not secure" warning on their phone but I wasn't able to replicate it, so I ignored it. Today, I started getting it on my phone as well. So I took the time to make the site as secure as possible and got the A+ rating on SSL Labs, but the warning won't go away. https://i.imgur.com/9Lz7Irs.png

I don't think it's possible to manually update every single instance of http in Humhub to https, so I've been looking for a better method.

According to this: [https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/upgrade-insecure-requests](ur), I could fix it using a meta tag, but I haven't had success with meta tags as I explained in #3631.

Another solution I found here: [https://www.a2hosting.com/kb/security/ssl/secure-and-insecure-content-on-a-web-page](url) using .htaccess, but it didn't work for me. They also include another meta tag method.

**Comment by kesselb** [comment]
Thank you for reporting this issue  :+1: 

I will need your browser console log to investigate this issue.  Open your console, reload your page and/or do the action leading to this issue and copy/paste the log in this thread.

<details>
<summary>How to access your browser console (Click to expand)</summary>

# Chrome
- Press either CTRL + SHIFT + J to open the “console” tab of the Developer Tools.
- Alternative method:
    1. Press either CTRL + SHIFT + I or F12 to open the Developer Tools.
    2. Click the “console” tab.

# Safari
- Press CMD + ALT + I to open the Web Inspector.
- See Chrome’s step 2. (Chrome and Safari have pretty much identical dev tools.)

# IE9
1. Press F12 to open the developer tools.
2. Click the “console” tab.

# Firefox
- Press CTRL + SHIFT + K to open the Web console (COMMAND + SHIFT + K on Macs).
- or, if Firebug is installed (recommended):
    1. Press F12 to open Firebug.
    2. Click on the “console” tab.

# Opera
1. Press CTRL + SHIFT + I to open Dragonfly.
2. Click on the “console” tab.
</details>

**Comment by Ace2213** [comment]
Well, I ended up breaking the whole website irreparably while trying to implement pretty URLs. Now that I've set up everything again, the warning seems to be gone, both from my phone and those who have brought it to my attention.

The only thing I've done differently is that I was able get meta tags working, which I explained in #3631 and closed it. Either that or I had some misconfiguration going on. Either way, it's solved. If it pops up again, I'll reopen the issue.
